projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-column-docs/table-column-docs.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component } from '@angular/core';
+import { GoTableConfig } from 'projects/go-lib/src/public_api';
+import { TableDocsService } from '../../table-docs.service';
+
+@Component({
+  selector: 'app-table-column-docs',
+  templateUrl: './table-column-docs.component.html'
+})
+export class TableColumnDocsComponent {
+  constructor(private tableDocsService: TableDocsService) {}
+
+  componentBindings: string = `
+  @Input() field:     string;
+  @Input() title:     string;
+  @Input() width:     number;
+  @Input() sortable?: boolean;
+  `;
+
+  fieldExample: string = `
+  new GoTableConfig({
+    tableData: [
+      {
+        id: 0,
+        name: {
+          first: John,
+          last: Smith
+        }
+      }
+    ]
+  });
+  `;
+
+  basicExample_html: string = `
+  <go-table class="go-column go-column--100" [tableConfig]="tableConfig" tableTitle="Example Usage">
```

Comment:
We want to try to keep these examples as light as possible, would you mind removing the `class` property from the example?

Reply:
only reason I put it here is because it's not obvious that's how the table gets it width. I was under the impression we should show everything required to match the output, but I'll take it out.

Reply:
I understand where you're coming from. I think we're more trying to make the examples as clear and concise as possible. Less markup means it'll be easier to scan and digest. 🤷‍♂